automates/text_reading/src/main/resources/application.conf

```diff
@@ -94,7 +94,7 @@ apps {
 
 alignment {
   alignerType = "pairwisew2v"
-  w2vPath = "/local/path/to/vectors.txt" // a word embeddings file (e.g., GloVe) with this header: <len_of_vocab><space><embedding_size>, e.g., "6B 50" (no quotes); use of different sets of embeddings will require adjusting alignment similarity thresholds, e.g., commentTextAlignmentScoreThreshold
+  w2vPath = "/Users/alicekwak/repos/automates/automates/text_reading/vectors.txt" // a word embeddings file (e.g., GloVe) with this header: <len_of_vocab><space><embedding_size>, e.g., "6B 50" (no quotes); use of different sets of embeddings will require adjusting alignment similarity thresholds, e.g., commentTextAlignmentScoreThreshold
```

Comment:
@alicekwak - The version of the config file in the repo should not have a user-specific path...

Reply:
@adarshp, thank you! This is me and Alice practising opening pull requests (this is a draft with more changes coming) and we will make sure to keep the conf file free of use-specific paths.

Reply:
@maxaalexeeva got it, no worries :)